affiliated/affiliated_package_review_process.md

```diff
@@ -88,56 +105,67 @@ The review process will be overseen and coordinated by the individuals that are
+   similar to that of a journal editor (and hence the role name). However this analogy
+   is only partially accurate, because here the editor may add their views to the review,
+   unlike a journal editor who is often strictly impartial on the review itself.
+   The pool of available reviewers will be anyone who has an official role on
+   [the Astropy team](http://www.astropy.org/team.html) or anyone who has participated
+   in the Astropy project enough to be familiar with the project guidelines and requirements.
+   In some cases, it may be necessary to have multiple reviewers depending on the
+   expertise required, and if scientific or algorithmic expertise is needed which
+   is not available in the project. A reviewer can be requested from outside the
+   project at the editor's discretion.
+
+2. The editor sends out an email to the potential reviewer to find someone who is
+   willing to review the package, using the [template](#templatereviewrequest)
+   at the bottom of this document.
+   Emails should only be sent to the next potential reviewer(s) - not
+   many possible reviewers at the same time. The editor should let the potential
+   reviewer know that at this point, that the editor may also be doing a review and
+   may synthesize the results into a single review.
+
+3. Once the reviewer accepts, the editor sends the reviewer an email with detailed
+   review instructions (which includes a link to the
+   [review guidelines](https://github.com/astropy/project/blob/master/affiliated/affiliated_package_review_guidelines.md>)),
+   using the [template](#templatereviewinstructions) at the bottom of this document.
+
+4. The editor then leaves a message on the pull request proposing the affiliated package saying:
+
+   *Thank you for proposing this package as an affiliated package! I'm happy to
+   confirm that your package is now under review and we'll post the results of
+   the review here and on the mailing list.*
+
+5. Once the reviewer has finished, they send their review back by email to the
+   editor, who then does their own review if needed and then writes up a review
+   using the review template, which includes a table with badges
+   ([template](#templatereviewresult) at the bottom of this document).
+   At this point, the editor makes a decision to accept or reject the package
+   based on the results of the review. If no objections or concerns are raised,
+   the editor then posts the review including the final decision (accept or reject)
+   on the original pull request.
+
+   * If the package is rejected:
+       * The original author has a month to respond to any of the points in the review,
+         and the editor may decide to change the decision.
+       * If decision to reject stands after a month, the decision is posted to the pull request
+         and it is closed.
+   * If the package is accepted:
+       * The editor sends a reply to the astropy-dev email to publish the decision.
```

Comment:
Does this email have a template too? Would be nice to have one to be consistent.

Reply:
We can look back at the list how the last few packages were announced and use that as template. I imagine this will be a two line email, so we can do without a template for now.

Reply:
There is no existing template, but feel free to create one!

Reply:
Can't find any in astropy-dev list as example. I'll make something up and push out in the next commit.

Reply:
Did you? Or want to leave that to a separate PR?